### PR TITLE
fix-exception-running-task-did-not-find-monitor-with-the-id

### PR DESCRIPTION
**Pull request by JKL98ISR**

### CODE REVIEW

File: backend/deepchecks_monitoring/bgtasks/alert_task.py [hunk 1]
```diff
@@ -125,7 +125,8 @@ async def execute_monitor(
     ))
 
     if monitor is None:
-        raise ValueError(f'Did not find monitor with the id {monitor_id}')
+        logger.info('Did not find monitor with the id: %s', monitor_id)
```
Comment (shayts7) [inline backend/deepchecks_monitoring/bgtasks/alert_task.py:128]: maybe warning? Also printing the organization_id?
Also, how do we reach into a situation that we do not find monitor?

**Review by shayts7 — APPROVED** [review]
See comments inside

File: backend/deepchecks_monitoring/bgtasks/alert_task.py (hunk 1 above)
Comment (JKL98ISR) [reply backend/deepchecks_monitoring/bgtasks/alert_task.py:128]: it was deleted, no need for a warning imo

Comment (JKL98ISR) [reply backend/deepchecks_monitoring/bgtasks/alert_task.py:128]: Info is maybe even too much maybe

**Review by shayts7 — APPROVED** [review]

**Pull request merged**